lib/standard/list.js

```diff
@@ -4,6 +4,10 @@ var fs = require('fs')
 var path = require('path')
 var util = require('util')
 var ProgressBar = require('progress')
+var bb = require('bluebird')
+
+var json = require('../util/json')
+var validate = require('../util/validate')
```

Comment:
This file is required in the root of  `lib/standard/index.js` which in turn is required in the root `lib/index.js` to make sure that a simple `nodeschool-admin version` call doesn't load too many modules I put all the require statements on purse in the run method. The only "global" require statements are those generally required like `npmlog`, `path`. Other require statements should (for the sake of snappiness) preferably be loaded only when necessary. (Note: "ProgressBar" sneaked its way into the file header, shouldn't be there in my opinion)